### ISSUE TITLE
The evaluation total loss is missing

### ISSUE THREAD

**Issue by frankdeepl** [tensorflow/models#7099]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**: models/research/object_detection/legacy
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: 
NO
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: Anaconda
- **TensorFlow version (use command below)**: 13.1
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: 9.0
- **GPU model and memory**: TITAN Xp
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

`python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`

### Describe the problem
When I tried to plot the training loss and evaluation loss by using tensorboard at the same time, I found the evaluation total loss and clone loss are missing, I only can see training total loss and clone loss. Other types of losses are right.
![Screenshot from 2019-06-25 19-03-21](https://user-images.githubusercontent.com/30953107/60096759-5fc90e80-9784-11e9-9ebc-98b42d544411.png)

Is this anybody know how to resolve this problem?

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by frankdeepl** [comment]
![Screenshot from 2019-06-25 19-03-21](https://user-images.githubusercontent.com/30953107/60096736-4b851180-9784-11e9-8d0a-ee75bfdc26c0.png)

**Comment by ppwwyyxx** [comment]
As I have commented in some other places (https://github.com/tensorpack/tensorpack/issues/1189#issuecomment-493682913, https://github.com/tensorpack/tensorpack/issues/1004#issuecomment-445479837), there is no existing definition of "evaluation loss" for a R-CNN model. So unless you define it yourself in the code, you certainly cannot see any "evaluation loss" in tensorboard.

**Comment by frankdeepl** [comment]
Hi ＠ppwwyyxx,
Thanks for your explanation.
